packages/manager/apps/telecom-spare/package.json

```
@@ -0,0 +1,56 @@
+{
+  "name": "@ovh-ux/manager-telecom-spare-app",
+  "version": "1.0.0",
+  "private": true,
+  "scripts": {
+    "build": "webpack",
+    "dev": "webpack-dev-server",
+    "dev:watch": "yarn run dev",
+    "start": "lerna exec --stream --scope='@ovh-ux/manager-telecom-spare-app' --include-filtered-dependencies -- yarn run build",
+    "start:dev": "lerna exec --stream --scope='@ovh-ux/manager-telecom-spare-app' --include-filtered-dependencies -- yarn run dev",
```

```suggestion
    "start:dev": "lerna exec --stream --scope='@ovh-ux/manager-telecom-spare-app' --include-filtered-dependencies -- npm run dev --if-present",
```
Some modules now use `webpack` to build and don't have a `dev` script